### ISSUE TITLE
English or French Version for Syntax definition

### ISSUE THREAD

**Issue by popujol** [swico/www.swiss-qr-invoice.org#12]
It would be helpfull to have a French English or italian vesion of Syntaxdefinition der Rechnungsinformationen (S1) bei der QR-Rechnung

https://www.swiss-qr-invoice.org/downloads/qr-bill-s1-syntax-de.pdf 

Vielen Dank im Voraus.

Pierre-Olivier

**Comment by epsitec** [comment]
@popujol you are right. I'll see if Swico can provide support for translation work...

**Comment by epsitec** [comment]
Giancarlo Palmisani (Swico) confirmed that a French translation will be ready hopefully by end of July 2020.

**Comment by popujol** [comment]
That would be great, thank you very much for asking them.

**Comment by tigerfoot** [comment]
Sorry but I don't know why this issues was marked as closed, resolved. Trying to download 
https://www.swiss-qr-invoice.org/downloads/qr-bill-s1-syntax-fr.pdf doesn't work.

So the issue is still open no ?

**Comment by epsitec** [comment]
Indeed, https://www.swico.ch/fr/connaissances/normes-et-standards/factures-qr/#creation-validation-et-syntaxe still does not provide the translated document.

**Comment by xLUDIOx** [comment]
There is an English translation included in the Implementation Guidelines for the QR Bill.
Starting at page 52: 
[https://www.paymentstandards.ch/dam/downloads/ig-qr-bill-en.pdf](https://www.paymentstandards.ch/dam/downloads/ig-qr-bill-en.pdf)

**Comment by epsitec** [comment]
The French translation is now online too.

See Swico's website directly:
https://www.swico.ch/fr/connaissances/normes-et-standards/factures-qr/
